client/ayon_core/plugins/publish/help/validate_unique_subsets.xml

```diff
@@ -11,7 +11,11 @@ Multiples instances from your scene are set to publish into the same folder > pr
 
 ### How to repair?
 
-Remove the offending instances or rename to have a unique name.
+Remove the offending instances or rename to have a unique name. Also, please
+    check your product name templates to ensure that resolved names are
+    sufficiently unique. You can find that settings:
```

Comment:
_Admittedly_ as a user I wouldn't know 'what to do' at those settings to ensure the resolved uniqueness. Nothing critical - but if anyone gets this invalidation message I have a feeling we might start getting questions about this. Let's see what comes up.